resources/views/adm/mship/account/ban/index.blade.php

```diff
@@ -0,0 +1,125 @@
+@extends('adm.layout')
+
+@section('content')
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="box box-primary">
+                <div class="box-header">
+                    <h3 class="box-title ">
+                        All Local Bans
+                    </h3>
+                </div><!-- /.box-header -->
+                <div class="box-body">
+                    <table id="mship-roles" class="table table-bordered table-striped">
+                        <thead>
+                        <tr>
+                            <th>User Banned</th>
+                            <th>Banned By</th>
+                            <th>Date Issued</th>
+                            <th>Starts</th>
+                            <th>Ends</th>
+                            <th>Type</th>
+                            <th>Status</th>
+                            @if($_account->hasPermission("adm/mship/account/*/note/create"))
+                                <th>Note</th>
+                            @endif
+                            @if($_account->hasPermission("adm/mship/ban/*/modify"))
+                                <th>Modify</th>
+                            @endif
+                            @if($_account->hasPermission("adm/mship/ban/*/repeal"))
+                                <th>Repeal</th>
+                            @endif
+                        </tr>
+                        </thead>
+                        <tbody>
+                        @foreach($bans as $b)
+                            <tr>
+                                <td>
+                                    {!! link_to_route("adm.mship.account.details", $b->account->name, [$b->account]) !!}
```

Comment:
Since this is a brand new file, should definitely be using single quotes for function arguments throughout, unless necessary for technical reasons).